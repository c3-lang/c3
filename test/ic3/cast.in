quote (U8) 256
quote (U16) 65536
