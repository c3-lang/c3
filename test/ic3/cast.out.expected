(U8) 256
(U16) 65536
